.github/workflows/npm-ecdsa.yml

```diff
@@ -48,6 +48,13 @@ jobs:
           commit: ${{ github.sha }}
 
       - name: Publish package
+        if: github.ref == 'main'
```

Comment:
Do we need these `main` checks? We already have it handled by https://github.com/keep-network/keep-core/blob/b09fcc41f8211ef6ea8451ecc646b4e733a70207/.github/workflows/npm-ecdsa.yml#L4-L6

So the workflow will be executed only on push to `main`.

Reply:
It can be also executed manually on any branch, that's why I added this clause.